Gemfile.prod

```diff
@@ -1,5 +1,3 @@
-ruby '~> 2.3.0'
```

Comment:
Currently we are using `2.3.1p112` in prod. Removing this will force us to upgrade before the next deploy. Are we sure we want to do that?